test_unstructured/partition/html/test_partition.py

```diff
@@ -618,48 +660,49 @@ def test_nested_text_tags(opts_args: dict[str, Any]):
         "  </p>\n"
         "</body>\n"
     )
-    opts = HtmlPartitionerOptions(**opts_args)
-    elements = HTMLDocument.load(opts).elements
 
-    assert len(elements) == 1
+    elements = partition_html(text=html_text)
+
+    assert elements == [NarrativeText("There is some text here.")]
 
 
-def test_containers_with_text_are_processed(opts_args: dict[str, Any]):
-    opts_args["text"] = (
-        '<div dir=3D"ltr">Hi All,<div><br></div>\n'
+def test_containers_with_text_are_processed():
+    html_text = (
+        '<div dir=3D"ltr">Hi All,\n'
+        "  <div><br></div>\n"
         "  <div>Get excited for our first annual family day!</div>\n"
-        '  <div>Best.<br clear=3D"all">\n'
```

Comment:
Was this supposed to have changed?

Reply:
Hmm, definitely didn't _need_ to change. I don't know why it did. `=3D` is a synonym for `=` and is only used in email as far as I know. I could add it back if you want. Otherwise it's a random test that both variants work :)